3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -12,8 +12,14 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    if ((number >= 10) & (number <= 99)) {
```

Comment:
Read up again about logical operators: a single `&` does a **bit-wise** AND (only used for very special purposes) but here you need a **logical** AND: `&&`.

Note that you don't need the extra parentheses, although it is not an error to use them here.

```suggestion
    if (number >= 10 && number <= 99) {
```